fdbserver/MoveKeys.actor.cpp

```diff
@@ -1521,7 +1521,7 @@ void seedShardServers(Arena& arena, CommitTransactionRef& tr, std::vector<Storag
 	tr.read_conflict_ranges.push_back_deep(arena, allKeys);
 	KeyBackedObjectMap<UID, StorageMetadataType, decltype(IncludeVersion())> metadataMap(serverMetadataKeys.begin,
 	                                                                                     IncludeVersion());
-	StorageMetadataType metadata(timer_int());
+	StorageMetadataType metadata(uint64_t(g_network->timer() * 1e9));
 
 	for (auto& s : servers) {
```

Comment:
I think the same change needs to be down [here](https://github.com/apple/foundationdb/pull/6389/files#diff-255bc4143a27b5f41ae87a1b44acdab6d2adbf7303420d9840bcd59788b9a7ebR1153-R1155)